_posts/2020-09-15-solidity-underhanded-contest.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+published: true
+title: 'The Solidity Underhanded Contest is back!'
+date: '2020-09-18'
+author: Franziska Heintel
+category: Announcements
+---
+
+We're excited to share that the [Solidity Underhanded Contest](https://underhanded.solidity.rocks/) is finally back!
+
+Inspired by the [Underhanded C Contest](http://www.underhanded-c.org/) and the first [Underhanded Solidity Contest](https://u.solidity.cc/), organized in 2017 by Nick Johnson, we decided it is time for a much needed revival.
+
+![Solidity Underhanded Contest](/img/2020/09/Underhanded_Solidity.gif)
+
+The goal of this contest is to write innocent-looking Solidity code, which pretends to clear and straightforward, but actually contains malicious behavior or backdoors.
+
+By hosting such a contest we aim to:
+
++ Raise awareness about smart contract security.
++ Uncover language design faults and backdoors.
++ Highlight anti-patterns in smart contact development.
++ Establish new best practices for secure smart contract development.
+
```

Comment:
Not sure about the exact formulation, but something like "battle-test newly introduced language features and restrictions'